src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -378,37 +378,25 @@ def WriteAttribute(self, nodeid: int, attributes):
             raise self._ChipStack.ErrorToException(res)
         return future
 
-    def ReadAttribute(self, nodeid: int, attributes: typing.List[typing.Tuple[int, ClusterAttribute.AttributeReadRequest]]):
+    def ReadAttribute(self, nodeid: int, attributes: typing.List[typing.Union[typing.Tuple[int, ClusterAttribute.AttributeReadRequest], ClusterAttribute.AttributePath]]):
```

Comment:
This makes for a very confusing API. Do we really need two ways to specify this?

I'd like to avoid consumers having to manually populate `ClusterAttribute.AttributePath` if we could, since that doesn't leverage any code-gen assistance.

Ideally, I think something like this might be more usable:

`ReadAttribute(1, [ (*) ])` => Read all endpoints.
`ReadAttribute(1, [ (1) ])` => Read all attributes from all clusters from EP1.
`ReadAttribute(1, [ (1, Clusters.TestCluster.Attributes) ])` => Read all attributes from the test cluster on EP1
`ReadAttribute(1, [ (1, Clusters.TestCluster.Attributes.AttributeA) ])` => Read attributeA from the test cluster on EP1.